package.json

```diff
@@ -35,23 +35,22 @@
     "@angular/cdk": "2.0.0-beta.8"
   },
   "devDependencies": {
-    "@angular/common": "^4.0.0",
     "@angular/animations": "^4.0.0",
-    "@angular/core": "^4.0.0",
-    "@angular/forms": "^4.0.0",
-    "@angular/platform-browser": "^4.0.0",
-    "rxjs": "^5.0.1",
-    "zone.js": "^0.8.14",
     "@angular/cli": "1.3.0",
+    "@angular/common": "^4.0.0",
     "@angular/compiler": "^4.0.0",
     "@angular/compiler-cli": "^4.0.0",
+    "@angular/core": "^4.0.0",
+    "@angular/forms": "^4.0.0",
     "@angular/http": "^4.0.0",
     "@angular/language-service": "^4.0.0",
+    "@angular/platform-browser": "^4.0.0",
     "@angular/platform-browser-dynamic": "^4.0.0",
     "@angular/router": "^4.0.0",
     "@types/jasmine": "~2.5.53",
     "@types/jasminewd2": "~2.0.2",
     "@types/node": "~6.0.60",
+    "attr-accept": "^1.1.0",
```

Comment:
is it necessary to depend `attr-accept`, there are only 36 lines of code in https://github.com/okonet/attr-accept/blob/master/src/index.js

Reply:
I think it should be added, sir. It looks clearer.
The 'attr-accept' was also introduced into the Upload component of Ant-Design.
Of course, if you think the inline function is better, I'm pleased to make the change.

Reply:
@thegatheringstorm it would better to not depend on unnecessary package since it would bring the risk of environment problem.

Reply:
@vthinkxie I'm sorry for the late reply.
Sure,there are only 36 lines of code in [attr-accept](https://github.com/okonet/attr-accept/blob/master/src/index.js).But...but it introduce [core-js](https://github.com/zloirock/core-js) in its code.
Should I add [core-js](https://github.com/zloirock/core-js) to my code?

Reply:
@thegatheringstorm 
both 'some' and 'endsWith' are supported by typescript.

Reply:
Thanks!